compiler-core/src/build.rs

```diff
@@ -234,14 +234,17 @@ impl Module {
         self.ast.find_node(byte_index)
     }
 
-    pub fn attach_doc_and_module_comments(&mut self) {
-        // Module Comments
-        self.ast.documentation = self
-            .extra
+    pub fn get_module_comments(&self) -> Vec<EcoString> {
+        self.extra
             .module_comments
             .iter()
             .map(|span| Comment::from((span, self.code.as_str())).content.into())
-            .collect();
+            .collect()
+    }
+
+    pub fn attach_doc_and_module_comments(&mut self) {
```

Comment:
not sure if it would make sense to just call this on each module when the lsp compiles or not